app/models/message.rb

```diff
@@ -0,0 +1,5 @@
+class Message
+  include ActiveModel::Model
```

Comment:
if all you're using this for is validations then you can just include ActiveModel::Validations, you should test this though because I'm not exactly sure what else you're using the message class for.